consensus/parlia/parlia.go

```diff
@@ -263,6 +267,10 @@ func New(
 	if err != nil {
 		panic(err)
 	}
+	recentHeaders, err := lru.NewARC(inMemorySignatures)
```

Comment:
inMemorySignatures = 256, is too small, double sign would trace back to recent 86400 blocks.
better to set the cache size to 86400